modules/300-prometheus/docs/FAQ.md

```diff
@@ -710,3 +710,64 @@ metadata:
    * You can only expand a PVC if its storage class's `allowVolumeExpansion` field is set to true.
 2. If storage doesn't support online resize, the message `Waiting for user to (re-)start a pod to finish file system resize of volume on node.` will appear in the PersistentVolumeClaim status.
 3. Restart the Pod to complete the file system resizing.
+
+## How to get information about alerts in a cluster?
+
+Run the following command to get cluster alerts:
```

Comment:
```suggestion
You can get information about active alerts not only in the Grafana/Prometheus web interface but in the CLI. This can be useful if you only have access to the cluster API server and there is no way to open the Grafana/Prometheus web interface.

Run the following command to get cluster alerts:
```